.rubocop.yml

```diff
@@ -26,6 +27,8 @@ Metrics/ClassLength:
     - 'app/controllers/surveys_controller.rb'
     - 'app/controllers/campaigns_controller.rb'
     - 'app/controllers/courses_controller.rb'
+    - 'lib/analytics/histogram_plotter.rb'
```

Comment:
Did you mean to exclude histogram plotter from the LineLength cop? This one applies to ClassLength.

Reply:
Nevermind, I see.

Reply:
So sorry, my fault. It is class length exception not a module length exception. :sweat_smile: